app/helpers/step_nav_actions_helper.rb

```diff
@@ -29,4 +30,12 @@ def can_take_over_review?(step_by_step_page, user)
       step_by_step_page.reviewer_id != user.uid &&
       user.has_permission?("2i reviewer")
   end
+
+  def links?(step_by_step_page)
```

Comment:
This is fine, but I think `contains_links?` or `has_links?` would be a little clearer.